### PR TITLE
Whoops

### PR DESCRIPTION
**Pull request by jaderabbit**
Cool, yeah - let's throw this one in

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -20,16 +21,16 @@ web:
...
-    GCP_PRIVATE_KEY: "__filler__ note keep the double quotes to make sure the private key is escaped"
-    GCP_PROJECT_NAME: __filler__
-    GCP_BUCKET_NAME: __filler__
+    SMTP_SENDER_NAME: Deep Learning Indaba
+    SMTP_SENDER_EMAIL: baobab@deeplearningindaba.com
+    SMTP_HOST: __filler__
+    SMTP_PORT: 587
+    GCP_CLIENT_ID: 114966260410024802375
+    GCP_CLIENT_EMAIL: baobab-service@baobab.iam.gserviceaccount.com
+    GCP_PRIVATE_KEY_ID: 2dafcfc720fb7b5d8d9248c58f574621a6861655
```
Comment (avishkar58) [inline docker-compose.yml]: Is this stuff meant to be here?

Comment (jaderabbit) [reply docker-compose.yml]: oh hell no

File: api/app/invitationletter/api.py [hunk 2]
```diff
@@ -29,18 +29,18 @@ def post(self):
         # Process arguments
         args = self.req_parser.parse_args()
         event_id = args['event_id']
-        work_address = args['work_address']
+        work_address = args['work_address'] if args['work_address'] is not None else ' '
```
Comment (KaleabTessera) [inline api/app/invitationletter/api.py]: I never knew you could do this :0

Comment (jaderabbit) [reply api/app/invitationletter/api.py]: python ternary :)

**Pull request merged**